tests/examples/test_disease_model.py

```diff
@@ -34,8 +34,24 @@ def test_disease_model(fuzzy_checker: FuzzyChecker, disease_model_spec: Path) ->
         "age",
         "sex",
         "entrance_time",
+        "simulant_step_size",
```

Comment:
We get a whole bunch of pipelines for free now that previously were "just" columns.